src/missing_deps.h

```diff
@@ -94,12 +68,14 @@ struct MissingDependencyScanner {
   int missing_dep_path_count_;
 
  private:
-#if __cplusplus >= 201103L
-  using EdgeAdjacencyMap = std::unordered_map<EdgePair, bool>;
+#if (__cplusplus >= 201103L) || (_MSC_VER >= 1900)
```

Comment:
`_MSC_VER >= 1900` shouldn't be needed as MSVC sets `__cplusplus` correctly with `/Zc:__cplusplus`.

Reply:
ack